gammapy/scripts/config/schema.yaml

```diff
@@ -44,47 +44,161 @@ properties:
     observations:
         description: "Observations used in the analysis."
         type: object
+        default: {}
         additionalProperties: false
         properties:
-            data_store:
-                description: "Data store where to fetch the observations."
+            datastore:
+                description: "Data store where to fetch observations."
                 type: string
-            filter:
+                default: $GAMMAPY_DATA/hess-dl3-dr1/hess-dl3-dr3-with-background.fits.gz
+            filters:
+                description: "Array of filters with criteria to select observations."
                 type: array
-                items: {"$ref": "#/definitions/observations_filter"}
+                default: [{}]
+                items:
+                    type: object
+                    additionalProperties: false
+                    properties:
+                        exclude: {type: boolean}
+                        inverted: {type: boolean}
+                        filter_type:
+                            default: par_value
+                            enum: [sky_circle,
+                                   angle_box,
+                                   par_box,
+                                   par_value,
+                                   ids,
+                                   all]
+                        frame:
+                            default: icrs
+                            enum: [galactic, equatorial, icrs, fk5]
+                        lon: {type: number, default: 83.633 deg}
+                        lat: {type: number, default: 22.014 deg}
+                        radius: {type: number, default: 0.5 deg}
+                        border: {type: number, default: 0.5 deg}
+                        variable: {type: string, default: TARGET_NAME}
+                        par_value: {type: string, default: Crab}
+                        value_range:
+                            type: array
+                            items: {type: number}
+                            minItems: 2
+                            maxItems: 2
+                        obs_ids:
+                            type: array
+                            items: {type: integer}
+                            default: []
+                    required: [filter_type]
+        required: [datastore]
 
-    grid:
+    reduction:
+        description: "Process of data reduction."
         type: object
+        default: {}
```

Comment:
Do we need these `default: {}` at intermediate levels?
I don't think that default empty dict is ever used, because we have defaults for sub-parts.
Remove here and in similar places?

Reply:
We need that when there are nested properties at a deeper level that have default values, it is a way to declare empty data structures.